cmd/lncli/invoicesrpc_active.go

```diff
@@ -0,0 +1,67 @@
+// +build invoicesrpc
+
+package main
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/lightningnetwork/lnd/lnrpc/invoicesrpc"
+	"github.com/urfave/cli"
+)
+
+// invoicesCommands will return nil for non-invoicesrpc builds.
+func invoicesCommands() []cli.Command {
+	return []cli.Command{subscribeSingleInvoiceCommand}
+}
+
+func getInvoicesClient(ctx *cli.Context) (invoicesrpc.InvoicesClient, func()) {
+	conn := getClientConn(ctx, false)
+
+	cleanUp := func() {
+		conn.Close()
+	}
+
+	return invoicesrpc.NewInvoicesClient(conn), cleanUp
+}
+
+var subscribeSingleInvoiceCommand = cli.Command{
```

Comment:
Do we need this `cli` edition? We don't have cli versions of the existing subscription type RPC's, and it's an odd use case IMO to watch invoices on the command line, as then you'd need some other system (automated or not) to do anything with the response. Also less code if we get rid of it all together.

Reply:
The reason I wrote it, is primarily to test the new rpc. I can imagine a user using this call instead of polling `lookupinvoice` on the command line. This use case wouldn't involve another system, but just giving the user a direct notification when the invoice is paid.

@cfromknecht what is your opinion on this cli command?

Reply:
Removed